generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -5,12 +5,16 @@
     kill_vm_on_error = yes
     login_timeout = 240
     reboot_count = 25
-    slots_mem = 4
-    maxmem_mem = 32G
     setup_hugepages = yes
-    mem_devs = hugepage
-    size_mem_hugepage = 256M
-    backend_mem = memory-backend-file
```

Comment:
For the previous scenario, delete it directly?
no tests of the previous scenario?

it's better to add previous scenario as a variant.